rpcs3/Emu/System.cpp

```diff
@@ -1115,6 +1119,9 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		// Initialize progress dialog
 		g_fxo->init<named_thread<progress_dialog_server>>();
 
+		// Initialize performance sensor
```

Comment:
Comment mismatches the class name now. 
I doubt it's really necessary, because the code is self-explanatory.